board.rb

```diff
@@ -2,67 +2,70 @@
 require_relative "cell"
 
 class Board
-  attr_reader :board, :status, :decease
+  attr_reader :status
   def initialize(rows = 8, cols = 8)
     @rows = rows
     @cols = cols
     @status = :alive
     @generations = Array.new
     @births = Array.new
     @deceases = Array.new
-    @board = Array.new(rows) { |i|
-      Array.new(cols) { |j|
-        Cell.new([i,j], rand(0..1).to_i)
-      }
-    }
+    @board = create_board
+  end
+
+  def create_board
+    Array.new(@rows) { Array.new(@cols) { Cell.new } }
   end
 
-  def iterationOf(gen)
-    temp = @board.map(&:clone)
-    iter_neighbors = 0
-    cells_alive = 0
-    deceases = 0
-    rebirths = 0
+  def iterationOf
```

Comment:
Remember to use snake_case instead of camelCase when naming methods or variables.